test/core/GnosisSafe.ModuleManager.spec.ts

```diff
@@ -244,4 +244,47 @@ describe("ModuleManager", async () => {
             ).to.be.deep.eq([false, "0x08c379a000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000013536f6d652072616e646f6d206d65737361676500000000000000000000000000"])
         })
     })
+
+    describe("getModulesPaginated", async () => {
```

Comment:
Do we have a test for the case that this method is called when no modules are enabled?